src/Dialog.js

```diff
@@ -44,6 +44,7 @@ class Dialog extends Component {
         return (
             <View style={[{
                 width: '100%',
+                flex: -1,
```

Comment:
Try to replace `flex: -1` to `flexShring: 1`

```suggestion
                flexShring: 1,
```